src/main/java/athena/commands/UnmarkCommand.java

```diff
@@ -0,0 +1,42 @@
+package athena.commands;
+
+import athena.exceptions.InputErrorCode;
+import athena.exceptions.InputException;
+import athena.tasks.TaskList;
+import athena.ui.Ui;
+
+/**
+ * Represents an unmark command given to Athena by the user. When executed, marks
+ * the task corresponding to the given task number as not done.
+ */
+public class UnmarkCommand extends Command {
```

Comment:
Perhaps you can try combining Mark and Unmark command into a single command since both are quite similar to reduce duplicated codes